packages/vite/scripts/generateSelfsignedCertificate.ts

```diff
@@ -0,0 +1,80 @@
+import fs from 'fs'
+import path from 'path'
+// @ts-ignore
+import { generate } from 'selfsigned'
+
+/**
+ * https://github.com/webpack/webpack-dev-server/blob/master/lib/utils/createCertificate.js
```

Comment:
this link does not work, createCertificate.js does not exist in the webpack-dev-server repo current master.

Skimming the repo it looks like they only offer configuration options for bringing your own cert, no auto- or pregenerated certs

Reply:
We were talking with @userquin that it may also be better for us to do the same. Maybe in Vite 3.0 we can remove the autogenerated cert to push users to create secure certificates